### PR TITLE
Smallpatches

### PR DESCRIPTION
**Pull request by InfusOnWoW**

### CODE REVIEW

**Comment by emptyrivers** [comment]
All of it looks good to me. The performance improvement for #1185 will introduce some churn, but it's probably not worth worrying about.

File: WeakAurasTemplates/TriggerTemplatesData.lua [hunk 1]
```diff
@@ -4209,15 +4586,12 @@ end
...
 -------------------------------
 
---[[
+
 for _, class in pairs(templates.class) do
   for _, spec in pairs(class) do
-  -- TODO 8.0
-  -- tinsert(spec[5].args, { spell = 195710, type = "ability"}) -- Honorable Medallion
-  -- tinsert(spec[5].args, { spell = 208683, type = "ability", pvptalent = 1}) -- Gladiator's Medallion
+    tinsert(spec[7].args, { spell = 208683, type = "ability", pvptalent = 1}) -- Gladiator's Medallion
```
Comment (mrbuds) [inline WeakAurasTemplates/TriggerTemplatesData.lua]: This doesn't test if `spec[7]` exists which make an error with current master for templates.class.ROGUE.classic

File: WeakAurasTemplates/TriggerTemplatesData.lua [hunk 2]
```diff
@@ -1518,6 +1643,11 @@ templates.class.PRIEST = {
     [7] = {
       title = L["PvP Talents"],
       args = {
+        { spell = 197871, type="ability", pvptalent = 5, titleSuffix = L["cooldown"]},-- Dark Archangel
```
Comment (mrbuds) [inline WeakAurasTemplates/TriggerTemplatesData.lua]: you can add `buff = true` to lines with `type="ability"` to give more choices

Comment (InfusOnWoW) [reply WeakAurasTemplates/TriggerTemplatesData.lua]: I know there are a bunch of things I could have done more for pvp talents, but I already spent far too much on this feature. And there are lots of small issues in there, as I didn't test them all.

**Comment by InfusOnWoW** [comment]
Can't we disable the templates for classic? I don't think we should release them anyway.

**Comment by Stanzilla** [comment]
If some commits are problematic for classic, we can just split the PR?

**Comment by mrbuds** [comment]
I'm not against disabling templates for classic until they are more refined

**Pull request merged**